.husky/pre-commit

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+set +e
+
+# Stash un-staged changes - optional, see note below
```

Comment:
Nit: I don't see any note below for these two comments, but I see why we would stash, lint, then un-stash